pallets/msa/src/offchain_storage.rs

```diff
@@ -258,8 +258,8 @@ pub enum IndexedEvent<T: Config> {
 		/// The key no longer approved for the associated MSA
 		key: T::AccountId,
 	},
-	/// Checks all existing keys and removes the wrong indexed ones
-	ReIndex {
+	/// Validates all existing keys and removes the wrong indexed ones
```

Comment:
nit: the event doesn't do the evaluation; it's merely a signal
```suggestion
	/// The offchain MSA->PubKey index has been marked invalid for the indicated MessageSourceId
```